### PR TITLE
Smb2

### PR DESCRIPTION
**Pull request by mauropalumbo75**
I added two events to the smb2 analyzer:

smb2-read-response
smb2-write-response

These events are useful for example to detect and log a file transferred in a single read or write response packet.

### CODE REVIEW

File: src/analyzer/protocol/smb/smb2-com-read.pac [hunk 1]
```diff
@@ -55,6 +55,13 @@ refine connection SMB_Conn += {
 			                 bro_analyzer()->Conn(), h->is_orig());
 			}
 
+		if ( smb2_read_request )
+			{
+			BifEvent::generate_smb2_read_response(bro_analyzer(),
```
Comment (sethhall) [inline src/analyzer/protocol/smb/smb2-com-read.pac]: You are generating a read_response message but checking above for a read_request.  You should be checking for the same event that is being generated in the if statement.

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
 build
 tmp
 *.gcov
+.idea*
```
Comment (sethhall) [inline .gitignore]: Was this an accidental addition?

**Comment by 0xxon** [comment]
Hi @mauropalumbo75 - I just wanted to ping on this and check if you saw the feedback of Seth :)

File: src/analyzer/protocol/smb/smb2-com-read.pac (hunk 1 above)
Comment (mauropalumbo75) [reply src/analyzer/protocol/smb/smb2-com-read.pac]: correct

File: .gitignore (hunk 2 above)
Comment (mauropalumbo75) [reply .gitignore]: yes

**Comment by mauropalumbo75** [comment]
Apologies for my late reply. 

Seth has definitely a point in his remarks and I actually came up with a solution to our issue at the script level without introducing new events. I push this alternative change and an additional test in https://github.com/mauropalumbo75/zeek/tree/smb2-fix and pulled another request.

**Pull request closed without merging**